### PR TITLE
Add error handling for dynamic imports to libdeno

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

File: core/isolate.rs [hunk 1]
```diff
@@ -556,13 +565,18 @@ impl Future for Isolate {
...
-        Ok(Ready(Some((dyn_import_id, mod_id)))) => {
-          self.dyn_import_done(dyn_import_id, mod_id)?;
-          continue;
+      loop {
+        match self.pending_dyn_imports.poll() {
+          Ok(NotReady) | Ok(Ready(None)) => break,
+          Ok(Ready(Some((dyn_import_id, mod_id)))) => {
+            match self.mod_evaluate(mod_id) {
+              Ok(()) => self.dyn_import_done(dyn_import_id, Ok(mod_id))?,
+              Err(..) => self.dyn_import_done(dyn_import_id, Err(None))?,
```
Comment (ry) [inline core/isolate.rs]: I’ve never seen this “..” before. That’s the same as “_” I guess ?

File: core/libdeno/exceptions.cc [hunk 2]
```diff
@@ -200,6 +200,7 @@ void HandleException(v8::Local<v8::Context> context,
   std::string json_str = EncodeExceptionAsJSON(context, exception);
   CHECK_NOT_NULL(d);
   d->last_exception_ = json_str;
+  d->last_exception_handle_.Reset(isolate, exception);
```
Comment (ry) [inline core/libdeno/exceptions.cc]: I wonder if there will be problems with the ASAN test. We might have to reset the handle in the destructor if so.

File: core/isolate.rs (hunk 1 above)
Comment (piscisaureus) [reply core/isolate.rs]: practically speaking, yes

**Comment by ry** [comment]
LGTM

File: core/libdeno/exceptions.cc (hunk 2 above)
Comment (piscisaureus) [reply core/libdeno/exceptions.cc]: Fixed -- added Reset() to the destructor.

**Pull request merged**